cosmos/providers/databricks/plugin.py

```diff
@@ -12,7 +12,13 @@
 from airflow.plugins_manager import AirflowPlugin
 from airflow.providers.databricks.hooks.databricks import DatabricksHook
 from airflow.security import permissions
-from airflow.utils.airflow_flask_app import get_airflow_app
+
+try:
+    from airflow.utils.airflow_flask_app import get_airflow_app
```

Comment:
Same here

Reply:
added a note that this is needed for Airflow versions < 2.3.3. Tested that this is not needed for >= 2.3.3